R/depth.R

```diff
@@ -1,9 +1,8 @@
 #' Data Depths
 #'
-#' `step_depth` creates a *specification* of a recipe
-#'  step that will convert numeric data into measurement of
-#'  *data depth*. This is done for each value of a categorical
-#'  class variable.
+#' `step_depth()` creates a *specification* of a recipe step that will convert
+#' numeric data into measurement of *data depth*. This is done for each value of
```

Comment:
```suggestion
#' numeric data into a measurement of *data depth*. This is done for each value of
```

I think so? Or "measurement" -> "measurements"